src/Monthly_Release.py

```diff
@@ -40,7 +40,6 @@
        end_date - (datetime) end of our date range we will return for
 OUTPUT: list of track ids found in the time range from all of the user's artists
 """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
-@logfunc
```

Comment:
Why remove the decorators here? Seemed to me that the decorator is meant to be applied to every function, and since these are outside of a class, shouldn't they need the decorator assigned like this, rather than the quick way you have for class' internal methods?

Reply:
So the decorator didn't actually do anything in this instance. My decorator uses a logger that is either part of the class, or that is passed into the function. This is not part of a class with a logger, and it does not have a logger passed in since it is global. This functionality was already being planned to move into Misc_Features.py but was out of scope for this story so the only reason the file stuck around was because I needed to be able to call it directly.